stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/options/MessageOptionsView.kt

```diff
@@ -64,8 +63,7 @@ public class MessageOptionsView : FrameLayout {
     private fun configureMineMessage(configuration: Configuration, syncStatus: SyncStatus) {
         val iconsTint = configuration.iconsTint
 
-        // Uncomment when the feature will be deployed to our backend
-        // binding.replyTV.isVisible = true
+        binding.replyTV.isVisible = true
```

Comment:
If this is always visible, we could set it in the XML and not touch it from code

Reply:
Make sense 👍